src/app/pages/credentials/backup-credentials/forms/ssh-connections-form.component.ts

```diff
@@ -356,11 +357,12 @@ export class SshConnectionsFormComponent implements FormConfiguration {
     );
   }
 
-  resourceTransformIncomingRestData(wsResponse: any): any {
+  resourceTransformIncomingRestData(wsResponse: KeychainCredential): any {
```

Comment:
When creating a new SSH Connection, can't submit the form because the `Private Key` field gets the error `Not an Integer` even though it's a dropdown and has `Generate New` option selected and no other option is available. This bug might not be related to this ticket though. Just mentioning here in case it is.